web/github_hook.php

```diff
@@ -7,10 +7,18 @@
 $app_root = realpath(__DIR__ . '/../');
 $composer = $app_root . '/composer.phar';
 
+require $app_root . '/vendor/autoload.php';
+
 // Git variables
 $branch = 'master';
 $header = 'HTTP_X_HUB_SIGNATURE';
-$secret = parse_ini_file($app_root . '/app/config/config.ini')['github_key'];
+
+$config_file = $app_root . '/app/config/config.yml';
+if (!file_exists($config_file)) {
+    throw new \InvalidArgumentException(sprintf('File %s is missing. Run composer install.', $config_file));
```

Comment:
A couple of nits here:
* We keep one space between the not and condition, i.e. `(! file_exists($config_file))`.
* We tend to use print/echo over printf, so `"File {$config_file} is missing. Run composer install."`.